src/main/java/frc/robot/auto/Path.java

```diff
@@ -65,17 +65,17 @@ public Path(ArrayList<PathPoint> Points) {
             PathPoint previousPoint = points.get(i-1); // Next in processing
 
             double distanceBetween = 
-                point.getDistanceAlongPathMeters() - previousPoint.getDistanceAlongPathMeters();
+                point.distanceAlongPath - previousPoint.distanceAlongPath;
 
```

Comment:
This doesn't take into account the final speed, which will cause the robot to do weird speeds on points close to eachother